obspy/signal/regression.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------
+# Filename: regression.py
+#  Purpose: Python Module for (Weighted) Linear Regression
+#   Author: Thomas Lecocq
+#    Email: Thomas.Lecocq@seismology.be
+#
+# Copyright (C) 2017 Thomas Lecocq
+# --------------------------------------------------------------------
```

Comment:
Can you merge that information with the copyright header right below it? These two also kind of contradict each other.